### ISSUE TITLE
Controller doesn't work with RPCS3

### ISSUE THREAD

**Issue by raspin0** [RPCS3/rpcs3#11314]
Most of the buttons in my controller aren't correctly detected even though they work in other games just fine. 
Controller: speedlink sl 6535 bk
OS: Deepin 20.3

https://user-images.githubusercontent.com/39202451/147762616-8f29918c-c7ea-409e-9ae7-107e92e15589.mp4

**Comment by hughesjs** [comment]
Hey dude, I'll have a look into this if I get a second. It'd be really helpful if you could capture the `evdev` output for your device using `evemu`.

It's really easy: https://www.freedesktop.org/wiki/Evemu/

If you drop us that, I can take a look and try to work out if there's some sort of mapping issue going on

**Comment by Megamouse** [comment]
It's missing a log file as well

**Comment by raspin0** [comment]
> Hey dude, I'll have a look into this if I get a second. It'd be really helpful if you could capture the `evdev` output for your device using `evemu`.
> 
> It's really easy: https://www.freedesktop.org/wiki/Evemu/
> 
> If you drop us that, I can take a look and try to work out if there's some sort of mapping issue going on

Thanks man, logs below
[lol.zip](https://github.com/RPCS3/rpcs3/files/7841940/lol.zip)

**Comment by Megamouse** [comment]
Did you actually try to remap the non-working buttons?
The defaults are just based on some xbox controller.

Anyway, your evdev config seems to be wrong.
You're missing ABS_RX.
And all of your axis have a range from 0-255. This is wrong. Stick axis should always have a range from - to +.

That being said, I am no Linux user, so I have no idea how to fix this.
In any case, this doesn't seem to be an RPCS3 issue.

**Comment by raspin0** [comment]
Okayy, I finally got this working and it exactly was RPCS3 specific problem. 

> And all of your axis have a range from 0-255. This is wrong. Stick axis should always have a range from - to +.

Apparently my pad is very weird. This was a true saviour - https://wiki.rpcs3.net/index.php?title=Help:Controller_Configuration#Analog_sticks_not_centered_with_evdev_pad_handler

It was quite difficult to figure out, but I had to modify provided configuration to:
```
ABS_X: true
ABS_Y: true
ABS_Z: false
ABS_RX: true
ABS_RY: true
ABS_RZ: true
```
RPCS will freak out otherwise and assign every click to RZ+ 

My pad is currently fully functional, I guess this issue can be closed.
